aws/gemini-3e/main.tf

```diff
@@ -0,0 +1,96 @@
+module "gemini-3e" {
+  source          = "../network-primitives"
+  path_to_scripts = "../network-primitives/scripts"
+  network_name    = "gemini-3e"
+  bootstrap-node-config = {
+    instance-type      = var.instance_type
+    deployment-version = 1
+    regions            = var.aws_region
+    instance-count     = var.instance_count + 1
```

Comment:
why +1 here though ?

Reply:
instance count is `1` by default, it's increased to launch 2 nodes.

Reply:
can't we just increase the actual value of instance_count in the vars instead of doing an addition here ?

Reply:
sure, but then we will have 2 full nodes and 2 farmers and 2 domain nodes and eventually 2 domain bootstrap nodes. However i will change this to a map.